client/components/RatingsReviews/RatingsReviews.jsx

```diff
@@ -5,19 +5,44 @@ import RatingBreakdown from './RatingBreakdown.jsx';
 import AddReview from './AddReview.jsx';
 import ProductBreakdown from './ProductBreakdown.jsx';
 import SortReviews from './SortReviews.jsx';
+import axios from 'axios';
+
+const getMetaData = function (productId) {
+  let url = 'http://localhost:3000/proxy/api/fec2/hratx/reviews/meta?product_id=' + productId;
+  return axios.get(url)
+    .then((response) => {
+      return response;
+    })
+    .catch((err) => {
+      throw err;
+    });
+};
 
 const RatingsAndReviews = (props) => {
-  return (<div className='jcontainer'>
-    <SortReviews />
-    <ReviewList id={props.reviewId || 12012}/>
-    <div className='ReviewSideBar'>
-      <div className='SideBarContainer'>
-        <RatingSummary />
-        <RatingBreakdown />
-        <ProductBreakdown />
+  const [metaData, setMetaData] = useState([]);
+  if (metaData.length === 0) {
+    getMetaData(props.product_id)
+      .then((data) => {
+        setMetaData(data);
+      })
+      .catch((err) => {
+        throw err;
+      });
+    return (<div>Loading...</div>);
```

Comment:
I think we can still conditionally render the Loading div if needed, but we are essentially setting initial state upon mount here right? In this case, instead of detecting if meta data exists yet to decide to run our AJAX, maybe we should use the useEffect hook to abstract out this idea, so that the ajax always runs upon mount and upon re-render. 

Shoot me a msg if you want me to clarify my thoughts